<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDialog</class>
 <widget class="QDialog" name="SelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>397</width>
     <height>259</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <item>
     <widget class="QLabel" name="label_text">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Items&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView_options"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_buttons">
      <item>
       <widget class="QPushButton" name="pushButton_OK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sellectAll">
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_deSellectAll">
        <property name="text">
         <string>DeSelect All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>SelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>SelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
